server/lib/vertex_tryon_client.py

```
@@ -0,0 +1,110 @@
+
+    request = PredictRequest(endpoint=_endpoint())
+    request.instances.append(_build_instance(avatar_bytes, item_image_bytes))
+
+    def _predict() -> bytes:
+        client = PredictionServiceClient(
+            client_options={"api_endpoint": _api_endpoint()}
+        )
+        try:
+            response = client.predict(request=request, timeout=timeout_s)
+        except gax.ResourceExhausted as exc:
+            raise VertexTryOnError(f"vertex quota exhausted: {exc.message}") from exc
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)